recipes/basemap-data-hires/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+    name: basemap-data-hires
```

Comment:
Specfiles allow for to package from the same source/spec. Maybe conda could adopt that idea in the future to avoid creating to recipes when splitting a package.